debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcChangeEventSink.java

```diff
@@ -68,6 +68,26 @@ public JdbcChangeEventSink(JdbcSinkConnectorConfig config, StatelessSession sess
         final DatabaseVersion version = this.dialect.getVersion();
         LOGGER.info("Database version {}.{}.{}", version.getMajor(), version.getMinor(), version.getMicro());
     }
+    
+    public boolean processDelete(JdbcSinkRecord record, final CollectionId collectionId, 
+    		final Map<CollectionId, Buffer> upsertBufferByTable, final Map<CollectionId, Buffer> deleteBufferByTable) {
+        if (!config.isDeleteEnabled()) {
```

Comment:
I would put this out avoiding the return value since, as the method name suggests, it will process the delete.